### PR TITLE
Add Inazuma elven go chrono stone tonnerre

### PR DESCRIPTION
**Pull request by sauron159**
Adds Inazuma elven go chrono stone tonnerre.

*Automatically generated by PR builder*

### CODE REVIEW

File: inazuma-elven-go-chrono-stone-tonnerre/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Inazuma elven go chrono stone tonnerre"
```
Comment (TheKoopaKingdom) [inline inazuma-elven-go-chrono-stone-tonnerre/game.dat]: Use the capitalized English title.

File: inazuma-elven-go-chrono-stone-tonnerre/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+description = "jeux qui fonctionne mal il ya 1 bug bleu on peux pas progresser dans le scenario"
```
Comment (TheKoopaKingdom) [inline inazuma-elven-go-chrono-stone-tonnerre/game.dat]: This must be in English, you can get it from Wikipedia as detailed in the [contributing guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#metadata).

File: inazuma-elven-go-chrono-stone-tonnerre/game.dat [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000136C00"
+region = "Jpn"
```
Comment (TheKoopaKingdom) [inline inazuma-elven-go-chrono-stone-tonnerre/game.dat]: This isn't a valid region. A list of regions can be found in the [contributing guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#metadata) (Capitalization matters!).

File: inazuma-elven-go-chrono-stone-tonnerre/game.dat [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+release_date = "2017-07-11"
```
Comment (TheKoopaKingdom) [inline inazuma-elven-go-chrono-stone-tonnerre/game.dat]: The game was released March 27th, 2015.

**Comment by Schplee** [comment]
Closing due to more than 2 weeks old, and completely broken.

**Pull request closed without merging**